frappe/patches/v15_0/set_contact_full_name.py

```diff
@@ -0,0 +1,12 @@
+import frappe
+from frappe.contacts.doctype.contact.contact import get_full_name
+
+
+def execute():
+	"""Set full name for all contacts"""
+	for name, first, middle, last, company in frappe.get_all(
+		"Contact",
+		fields=["name", "first_name", "middle_name", "last_name", "company_name"],
+		as_list=True,
+	):
+		frappe.db.set_value("Contact", name, "full_name", get_full_name(first, middle, last, company))
```

Comment:
Enable auto-commit. I've seen sites with millions of contacts :smiling_face_with_tear: 

Converting to raw sql update might also be a better option.

Reply:
Also `update_modified=False`

Reply:
@ankush done. I agree, SQL would be ideal, but I don't want to think about the combination of `COALESCE`, `CONCAT` and `TRIM` needed to achieve exactly the same behavior as `get_full_name` right now. 😅🙈